regression.sh

```diff
@@ -106,8 +106,9 @@ if [ ${cur_count} != ${core_count} ]; then
     if [ ${core_count} -gt 0 ]; then
         for corefile in "${corefiles[@]}"
         do
-            executable_name=$(echo ${corefile} | awk -F'-' '{ print $1 }' \
-                              | cut -d'/' -f2)
+            executable_name=$(gdb -ex "core-file ${corefile}" -ex \
+                'set pagination off' -ex 'info proc exe' -ex q \
+                2>/dev/null | tail -1 | cut -d "'" -f2)
```

Comment:
Currently the output here is like,
"/build/install/sbin/glusterfsd -s slave26.cloud.gluster.org --volfile-id patchy"

Which when passed to line 111 (to which) fails.

I think a further cut is required a the end (like, "| cut -d ' ' -f 1") to just get the executable name. This cut gets us "/build/install/sbin/glusterfsd", which again fails in `which`, but may not in the regression machines. To be safer, a further grep (or alternative) maybe needed just to get the executable name, like so "| grep -o '[^/]*$'".